src/main/kotlin/skytils/skytilsmod/utils/ItemUtil.kt

```diff
@@ -229,4 +230,14 @@ object ItemUtil {
 
     fun getStarCount(extraAttributes: NBTTagCompound) =
         extraAttributes.getInteger("upgrade_level") + extraAttributes.getInteger("dungeon_item_level")
+
+    fun isSalvageable(stack: ItemStack): Boolean {
+        val extraAttr = getExtraAttributes(stack)
+        val sbId = getSkyBlockItemID(extraAttr)
+        return sbId != "ICE_SPRAY_WAND" && extraAttr != null && getStarCount(extraAttr) == 0
+                && !extraAttr.hasKey("dungeon_item") && (extraAttr.hasKey("baseStatBoostPercentage") || getItemLore(
```

Comment:
remove extra lore check as they added the nbt tag back, probably move the baseStatBoosPercentage up to the front